CHANGELOG.md

```diff
@@ -1,4 +1,10 @@
 ## [UNRELEASED]
+
+
+### Bug Fixes
+
+* **aws-cloudfront:** Fix empty string defaultRootObject being replaced with the default value instead of passing the empty string to CloudFormation. Closes [#794](https://github.com/awslabs/aws-cdk/issues/794)
```

Comment:
Not required. Just follow conventional commits and it will be added to the changelog

Reply:
https://github.com/awslabs/aws-cdk/blob/master/CONTRIBUTING.md says it is required :(